src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedTargetPlatforms.props

```diff
@@ -0,0 +1,21 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.SupportedTargetPlatforms.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project>
+  <ItemGroup>
+    <SupportedTargetPlatform Include="Android" />
```

Comment:
@terrajobst is this the right name or did we decide to go with `SupportedPlatform`?